### ISSUE TITLE
Appreciate for you

### ISSUE THREAD

**Issue by kmh4830** [asif-mahmud/MIFARE-RFID-with-AVR#3]
Hello. I'm trying to work with atmega128 and rc522.

i think that you are the only one who shares atmega rc522 source (no arduino)

so i really appreciate for you

would you mind giving a little hand for me?

I just tried this module 3 hours before

And I finally modified to atmega128 from your source.

but atmega128 is getting "Not Recognised !" message.

==
	uint8_t Blue[5]  ={0x3d,0x21,0xe4,0xd3,0x2b};
	uint8_t White[5] ={0x0e,0x65,0xe2,0x13,0x9a};
==
i think that two arrays mean certain card's id or address..?
so i need to verify that arrays to my card's id or address..

but i don't know my card's id.... !!

i'm trying to find my card's id

i guess there's got to be some part to extract detecting card's id in your source..
but i can't find it

please help me

**Comment by asif-mahmud** [comment]
Yes, it was for testing purposes. I had 2 cards at that time i guess :)

Best wishes -
Asif Mahmud Shimon
